charters/DataPortal/charter.md

```diff
@@ -0,0 +1,81 @@
+### General
+* UX, visual and branding design with input from HCA leadership (GG, scientific steering committee, etc.)
+* Participation in the HCA-wide standards for Continuous Integration (CI), Continuous Delivery (CD), system logging via Google Analytics, etc.
+* Production operation of the overall Data Portal web-site (see exclusions in Out-of-scope section)
+* Implementation performance scaling
+* DevSecOps compliance as required by the HCA Security Team
+* DevSecOps features for FISMA moderate deployments (authZ/N, logging, auditing, etc) by leveraging other projects (if this becomes a future requirement)
+* Community engagement
+   * Triage and integration of feature requests into the work queue 
+   * Support plan for escalation of user reported bugs
+### Data Portal
+* Select and deploy content management system
+* Integrate Data Browser and Data Portal UIs into a seamless UX and provide link to other groups' applications (e.g. Ingest UI)
+* Frequent updates of summary statistics (cells, organs, donors, projects, labs, etc.) on Home Page
+* Coordinate development of user guide content (e.g. “Learn”) from multiple contributors
+* Coordinate development of developer documentation content (e.g. “Develop”) from multiple contributors. Includes API documentation and development guides.
+* Click through to Data Browser from search box and human body image
+* Registration form for tertiary portals and list of portals available with click through to hosting web site (e.g. Dockstore)
+* Integration with Zendesk help desk
+* Integration with Vanilla Forums (discussion forum for developers)
+* Integration with Canny (to capture user feedback)
+### Data Browser
+* Create (or use Data Store provided) indexes as necessary for files in the Data Store, using public DSS APIs to read the metadata
+* Browser support for multiple metadata schemas that are supported by the schema access library maintained by the Ingest service
+* Search views of Specimens, Projects and Files
+* Browser UI to search the DSS by specifying multiple search criteria (“facets”)
+* Download selected files from the DSS via the UI and/or HCA cli (the HCA cli itself is out-of-scope for this charter)
+* AuthN for tracking users' collections, saved searches, etc.
+* Full authN/Z if the HCA stores controlled access data in the future. Leverage AuthN/Z from Data Store.
+* Browser interface with DSS Collections service to save/view/list/share/delete the query results from a search operation
+* Handoff of a Collection of search results to a tertiary portal. Provide for direct handoff of search results to FireCloud.
+* Save/view/share/update/delete search queries and personal collections
+* Search results handoff to Matrix service
+* API Documentation for portable, cloud-neutral APIs
+
+## Out-of-scope
+* Production operation of other HCA Data Coordination Platform (DCP) components in AWS (e.g. Data Store, Ingest, Secondary Analysis, Tertiary Analysis)
+* Contribute/Ingest UI (from EBI)
```

Comment:
Separate these items and give a couple words more context. If I were a new user, I would not know what either of these things were.